static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -112,15 +112,10 @@
     font-weight: 200;
 }
 
-.immersive-main-media__headline-container--dark {
-    .content__headline--immersive-article {
-        color: #ffffff;
-    }
-}
-
```

Comment:
Yeah that's fair enough, feel free to take a closer look at this.  The thinking for keeping my PR super low-impact was that there were already plans to potentially refactor some of this so tones/immersives wouldn't override each other in this way.  That's probably fairly long-term though, so shouldn't stop you from going ahead.